runtime/appruntime/api/server.go

```diff
@@ -88,11 +94,21 @@ func NewServer(
 	pc *platform.Client,
 	encoreMgr *encore.Manager,
 	rootLogger zerolog.Logger,
-	metrics *metrics.Manager,
+	reg *metrics.Registry,
 	json jsoniter.API,
 	tracingEnabled bool,
 	clock clock.Clock,
 ) *Server {
+	requestsTotal := metrics.NewCounterGroupInternal[RequestsTotalLabels, uint64](reg, "e_requests_total", metrics.CounterConfig{
+		EncoreInternal_LabelMapper: func(labels RequestsTotalLabels) []metrics.KeyValue {
+			return []metrics.KeyValue{
+				{Key: "service", Value: labels.Service},
+				{Key: "endpoint", Value: labels.Endpoint},
+				{Key: "code", Value: labels.Code},
+			}
+		},
+	})
```

Comment:
I haven't set `EncoreInternal_SvcNum` as this metric is shared by all services. My understanding is that the service number is read from the request tracker.

Reply:
That's the case where it's used.